ci_config.json

```diff
@@ -21,9 +21,13 @@
     "sdl2_net", "sdl2_ttf", "termbox", "unit-system", "xxhash", "zstd"
   ],
 
+  "fluidsynth": {
+    "linux_only": true
```

Comment:
linux_only should only be used for projects that cannot work on other platforms, for example when they are made for a specific Linux kernel module. Is it the case here?